### ISSUE TITLE
Any pointers to which part of the specification in implemented and which are still pending?

### ISSUE THREAD

**Issue by ahsanity** [steffenbrem/JsonApiBundle#20]
Hi there! Great work with this bundle! 

I understand this is still a WIP, but is there a doc (or checklist or reference somewhere in code, or where ever) that specifies which part of the specification is implemented, and what are still pending? It will help people (like me) to understand if we can use it in our projects (I understand that this is still not recommended for production), with an assessment of the risks. Also, this will give visibility to (wannabe) contributors on where they can start contributing, if anyone is interested. 

This checklist might also include the status of a particular spec segment like stable, buggy, not implemented. 

Just a thought.

**Comment by koemeet** [comment]
Yes, this is a good idea. It seems more people are interested in this bundle, not only me anymore. I will add those to the README.

**Comment by ahsanity** [comment]
That would be great. Thanks!

**Comment by andreasschacht** [comment]
Hi all,

i'm also very interested in this bundle. Are there some news to this topic?

Thanks!

**Comment by senkal** [comment]
Hi @steffenbrem .
Would be great if you could specify what are your goals for, lets say, production ready stage.
I am very interested into this plus can help you with development.
If you have anything in mind that should be done next, could help you with that.

I have some basic question after some investigation.
1. Do you plan to add tests, unit tests/functional?
2. Do you plan to support more configuration formats than annotations?
3. Any plans for integrating with Travis, other CI?

**Comment by RuslanZavacky** [comment]
@senkal I am currently covering (1), (2) and hopefully (3) in #34. And as a new functionality - Json Api Object deserialization.
